### PR TITLE
Feat/profile details mock data

### PR DESCRIPTION
**Pull request by rekesan**

### CODE REVIEW

**Review by jendan-nikolai — APPROVED** [review]

File: app/Login.tsx [hunk 1]
```diff
@@ -33,6 +31,9 @@ const KeyboardView = styled(KeyboardAvoidingView)({
   flex: 1,
   minHeight: 600,
   backgroundColor: "$bg",
+  width: "100%",
+  maxWidth: [null, null, 386],
```
Comment (mika-ella) [inline app/Login.tsx]: why is there null values?

File: app/_layout.tsx [hunk 2]
```diff
@@ -26,6 +26,88 @@ const theme = makeTheme({
     bold: "600",
     extraBold: "900",
   },
+  buttons: {
```
Comment (mika-ella) [inline app/_layout.tsx]: should be button only

File: app/header.tsx [hunk 3]
```diff
@@ -49,68 +31,46 @@ const Title = styled(Text)({
...
-    height: 38,
-    width: 38,
-    minWidth: 38,
-    borderRadius: 38 / 2,
-    backgroundColor: "$light",
-    flexDirection: "row",
-    flex: -1,
-  })
-
+const Header = ({ title, titleColor, style, ...props }: HeaderProps) => {
```
Comment (mika-ella) [inline app/header.tsx]: do not use spreading

File: app/profile/[userId].tsx [hunk 4]
```diff
@@ -82,47 +68,13 @@ const ProfileBioText = styled(Text)({ fontSize: 18, marginLeft: 15 });
 const Profile = () => {
   const { userId } = useSearchParams();
   const [user, setUser] = useState<UserProps>();
-  const sx = useSx();
 
   if (userId === "me") {
-    getUserLoggedIn().then((userLoggedIn) =>
-      getUserData({ username: userLoggedIn }).then((user) => setUser(user))
-    );
+    getUserDataLoggedIn().then((user) => setUser(user));
```
Comment (mika-ella) [inline app/profile/[userId].tsx]: add user interface inside the then function for proper readability

**Review by mika-ella — APPROVED** [review]

**Pull request merged**